webapps-2019.html

```diff
@@ -286,7 +286,7 @@ <h3>
                 An API for a database of records holding simple values and
                 hierarchical objects. The <a href=
                 "https://w3c.github.io/IndexedDB/">third edition</a> adds new
-                capabilities and developer ergonomic improvements by using
+                capabilities and improves developer ergonomics by using
                 promises.
```

Comment:
This might just be me not used to the terminology

Reply:
No, this is a great suggestion.